base/applications/taskmgr/lang/it-IT.rc

```diff
@@ -343,21 +343,21 @@ BEGIN
     ID_WINDOWS_CASCADE "Sovrappone diagonalmente le finestre sul desktop"
     ID_WINDOWS_BRINGTOFRONT "Porta la finestra in vista ma senza attivarla"
     ID_HELP_TOPICS "Visualizza gli aiuti di Task Manager"
-    ID_HELP_ABOUT "Visualizza le informazioni sul program, version e copyright"
-    ID_FILE_EXIT "Esce da Task Manager"
-    ID_OPTIONS_SHOW16BITTASKS "Mostra i task a 16-bit con ntvdm.exe associato"
+    ID_HELP_ABOUT "Visualizza le informazioni sul programma, versione e copyright"
+    ID_FILE_EXIT "Esce da Gestione attività"
+    ID_OPTIONS_SHOW16BITTASKS "Mostra i processi a 16-bit con ntvdm.exe associato"
     ID_VIEW_SELECTCOLUMNS "Seleziona quali colonne saranno visibili nella pagina dei processi"
     ID_VIEW_SHOWKERNELTIMES "Mostra il tempo di kernel nel grafico delle prestazioni"
     ID_VIEW_CPUHISTORY_ONEGRAPHALL "Un singolo grafico mostra l'uso di cpu totale"
     ID_VIEW_CPUHISTORY_ONEGRAPHPERCPU "Ogni CPU ha il suo grafico"
-    ID_APPLICATION_PAGE_SWITCHTO "Porta un task in vista e gli dà il focus"
+    ID_APPLICATION_PAGE_SWITCHTO "Porta un processo in vista e gli dà il focus"
 END
 
 STRINGTABLE
 BEGIN
-    ID_ENDTASK "Richiede la chiusura del task selezionato"
-    ID_GOTOPROCESS "Porta il focus al task selezionato"
-    ID_RESTORE "Ripristina Task Manager dallo stato nascosto"
+    ID_ENDTASK "Richiede la chiusura del processo selezionato"
+    ID_GOTOPROCESS "Porta il focus al processo selezionato"
+    ID_RESTORE "Ripristina Gestione attività dallo stato nascosto"
```

Comment:
You changed that string back to "Task Manager" in several places, but you forgot to do the same here?